CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## [next]
 
+### Added
+- Added environment variables: `LAN_ONLY` and `MAX_SNAPSHOTS`.
+
+### Changed
+- Adopt the new file structure based on clusters.
+- Rename environment variable: `SERVER_TOKEN` => `DST_TOKEN`.
```

Comment:
Whole sentence please, like "Rename environment variable `SERVER_TOKEN` to `DST_TOKEN`.".